index.d.ts

```diff
@@ -243,6 +243,11 @@ declare module 'warframe-items' {
         pellet?: Pellet;
         duration?: number;
         speed?: number;
+        falloff?: {
+            start: number;
+            end: number;
+            reduction: number;
+        };
```

Comment:
this should be broken out into another interface, shouldn't it? (at least from the pattern others have used on typings in here)

Reply:
Oh right, this was how I fixed it in my implementation, but maybe it would be better to break it out